Package.swift

```diff
@@ -126,6 +134,26 @@ let targets: [Target] = [
     swiftSettings: defaultSwiftSettings
   ),
 
+  // Retroactive conformances of gRPC client and server to swift-server-lifecycle's Service.
+  .target(
+    name: "GRPCServiceLifecycle",
```

Comment:
It'd be great if you could add a documentation bundle for this too (and update `.spi.yml`). That can be a follow up though if you prefer.

Reply:
Hm, looks like none of the modules other than the Health service actually have a doc bundle so maybe we should do that separately for all?

Reply:
Sounds good to me